### PR TITLE
Add approve-pls and reject-pls; fix a typo

### PR DESCRIPTION
**Pull request by makyen**

### CODE REVIEW

File: pages/faq.md [hunk 1]
```diff
@@ -205,6 +205,8 @@ be effective in reviewing the request.
     [tag:reopen-pls] post has enough info now https://stackoverflow/q/12345
     [tag:del-pls] has no value to stay on site https://stackoverflow/q/12345
     [tag:undel-pls] Adds value to site https://stackoverflow/q/12345
+    [tag:approve-pls] Important edit w/ reject https://stackoverflow.com/review/suggested-edits/12345
```
Comment (gunr2171) [inline pages/faq.md]: "Important edit with reject"? I don't understand that example.

**Comment by makyen** [comment]
I wasn't sure about explicitly allowing approve-pls requests for your own edits when removing plagiarism from tags. However, I've seen approve-pls used that way a few times by various people (even [doing so myself](https://chat.stackoverflow.com/transcript/41570?m=36508412#36508412) in April when a tag edit I made to remove plagiarism got a reject vote). Doing so specifically when the edit is removing plagiarism has appeared to be implicitly acceptable. In writing this section, I was attempting to match what I had seen to be accepted in the room (i.e. not change policy, just render it to text).

When I ended up posting such a request myself, I considered quite some time prior to choosing to post a request about my own edit. The primary argument I had for posting the request was that A) dealing with plagiarism on tag-wikis/tag-excerpts is specifically a moderation task; B) moderation is the focus of SOCVR; and C) SO users with tag review privileges are explicitly supposed to involve themselves with checking for and dealing with plagiarism in tag-wikis/excerpts (per [Undo's post about review bans for approving plagiarism on tag-wiki/excerpts](https://meta.stackoverflow.com/questions/318288/stopping-tag-wiki-plagiarism-part-ii-taking-action)). [Note: in my specific case, removing the plagiarism was also [requested in SOCVR by a different member](https://chat.stackoverflow.com/transcript/message/36496514#36496514).]

I'm not strongly in favor of the exception, but, IMO, the balance is that specific exception should be allowed. More importantly, such requests being acceptable appears to be what's currently implemented in the room. However, I can understand if we want to leave it as something that's not explicitly permitted.

If it would be preferable, I can re-write this without that exception and/or have the exception as a separate PR.

**Comment by gunr2171** [comment]
Ah, I understand what you're getting at.

This is something we _need_ to run by the room, but I'd be in favor of allowing `accept-pls` requests ONLY if:

1. It is a wiki tag edit
2. You are fixing plagiarism or some other seriously important change.

As for the written example, how about

    [tag:approve-pls] Removes plagiarism, has reject vote https://stackoverflow.com/review/suggested-edits/12345

Else, I'm fine with merging as-is.

**Comment by makyen** [comment]
#### Example
I'm fine with changing the example to whatever is desired. I like yours better than what I have. I was intentionally keeping it horizontally short due to assuming my preview was accurately indicating when horizontal scrolling would be used. It was not. On GitHub, there is enough space for what you suggest prior to scrolling (one additional character is possible without horizontal scrolling). On socvr.org, it's unclear if the width is a problem.

Unlike GitHub, socvr.org doesn't have a fixed width for code blocks. In addition, the way the CSS is now, there's no `overflow: auto;` on `<pre>` elements, so there is no horizontal, or vertical, scrolling. Unfortunately, this leads to long lines in code blocks extending outside of the `<div id="content>` .  However, if one assumes socvr.org is being viewed at the same window width as the minimum needed for Stack Overflow to display without scrolling, the line you've suggested doesn't exceed the `content` width.

I've updated the PR with the example you suggested.

#### CSS change: will submit another PR
I would recommend changing the CSS to both show code a bit more clearly and add `overflow: auto;` I'll submit a separate PR for that momentarily.

#### Yet another PR `https://stackoverflow/` to `https://stackoverflow.com/`:
I just noticed that we're using `https://stackoverflow/` not `https://stackoverflow.com/` on all the other examples. I would suggest changing it to `https://stackoverflow.com/` for all of them. I'll submit a separate PR for that. However, having it as a separate PR will probably result in a merge conflict as it would need to modify both the line above and below the example lines this PR is adding.

#### approve-pls
It's unclear to me if your statement about approve-pls is wrt. *all* approve-pls requests, or just for those which are about one's own edits. However, I'm fine either way. If you're talking about the restriction on approve-pls for one's own edits, I don't see any other circumstance where I would consider it acceptable beyond what you've detailed. Unless someone comes up with an example I haven't thought of, I'd argue for the same restrictions you've stated. In other words, a *very* limited circumstance, but one that has happened from time to time.

If you were talking about approve-pls requests for other people's edits: there's a corner case where if one has already "approved" the edit, you can't force the edit to be applied. You can force a reject after selecting approve or reject in the review queue by forcing your own edit of the post, but I don't believe you can force the current edit to be approved. Prior to approving/rejecting, anyone with edit privileges can easily force the resolution of a suggested edit in either direction by using reject and edit, or improve edit. Frankly, approve-pls requests should be quite rare.

#### Running it by the room
Running it by the room sounds like a good thing to do. What's the best way to go about doing so for this? Bring it up for discussion when there happens to be a reasonable number of regulars/ROs active? Add it as a topic for the next room meeting?

**Review by gunr2171 — APPROVED** [review]

**Comment by rschrieken** [comment]
@makyen  a topic on the next room meeting (6 to 8 weeks) should be fine.

**Pull request merged**